lib/metasploit/framework/password_crackers/cracker.rb

```diff
@@ -82,7 +86,7 @@ class Cracker
 
         validates :config, :'Metasploit::Framework::File_path' => true, if: 'config.present?'
 
-        validates :cracker, inclusion: {in: %w(john hashcat)} 
+        validates :cracker, inclusion: {in: %w(john hashcat)}
```

Comment:
`%w[]` for arrays of non-interpolated strings.

Reply:
https://github.com/rapid7/metasploit-framework/commit/27ea63ad2533e5f352ee7323ac262cbb9915a9b0